repository_service_tuf_api/__init__.py

```diff
@@ -29,6 +30,13 @@
 )
 
 
+@dataclass
+class BootstrapState:
+    bootstrap: bool
+    state: Optional[str] = None
```

Comment:
I think it does make sense to make the `state` as an Enum, so that it's clear which are the allowed values.
What do you think?

Reply:
I thought to have it. I even did it, and I reverted it.
I reverted because it will be inconsistent if we add more "`<state>-`" in the future.
If you see it now, we don't make decisions on the `state` level. We only use the `bootstrap`.

The state will be used in specific decisions to sign the bootstrap, which requires "`signing-`".
Isn't it better to leave the control to the logic in that case?

Reply:
> The state will be used in specific decisions to sign the bootstrap, which requires "`signing-`".
> Isn't it better to leave the control to the logic in that case?

We will still leave the control to the logic of the signing case and we won't use it.
We will just make it easier for the next implementation to not use a hardcoded `signing` string, but instead the Enum values.

Reply:
We can decide on it in the DAS implementation. There is no use case of `state` for this specific PR.